src/components/screens/Layout.vue

```diff
@@ -1,6 +1,13 @@
 <template>
   <rate-limit-screen v-if="isRateLimit" />
-  <router-view v-else />
+  <div v-else>
+    <portal-target
```

Comment:
Why we need portal instead route?

Reply:
because we should show container in full screen, not in v-frame